docs/tutorial/CustomDataLoaderOnSL.ipynb

```diff
@@ -231,18 +223,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "c4ee4435",
    "metadata": {},
    "source": [
-    "接下来我们将user、movie和rating进行join，并进行拆分，组装成`alice_ml1m.csv`和`bob_ml1m.csv`"
+    "Next, we join user, movie and rating. Split them and assemble them into alice_ml1m.csv and bob_ml1m.csv."
```

Comment:
`. Split` -> `, split`.
(`) which includes the name of the file should be retained.